### DISCUSSION TITLE
Generate Build Server Config with relative paths

### DISCUSSION THREAD

**Discussion by jonathan-guide** [sweetpad-dev/sweetpad#87] [Any discussions (ideas, questions, ...)]
Hi,

This tool is great! So great that I want to share it with my team members.

Is it possible to generate a buildServer.json that doesn't use my absolute path? I wouldn't want to commit absoluate paths like this to source control, but I would want to commit the file.

Any suggestions appreciated!

**Comment by NSExceptional** [comment]
+1, this is super annoying

**Comment by hyzyla** [comment]
> but I would want to commit the file.

i just recently started exploring what is possible todo here, could you expalain why you want to commit the file, intead of gitignoring it?

**Reply by NSExceptional** [reply]
I recently found out that SweetPad will generate it automatically as necessary (is that correct?) but I don't think that was always the case. Perhaps this discussion is from back when you had to generate it by running a command to do so before anything would work?

Anyway, git ignoring it is what I will be doing going forward for sure
